src/components/ImageGrid/ImageGridComponent.tsx

```diff
@@ -98,29 +72,100 @@ const ImageGridComponent = ({
     } else {
       setRowCol({ col: 1, row: 1 });
     }
-  }, [numberOfImgs, images]);
-
-  // to avoid bug undefined url when user change number of images
-  useEffect(() => {
-    for (let i = 0; i < numberOfImgs; i++) {
-      images[i] = exampleImages[i];
-    }
   }, [numberOfImgs]);
 
   return (
     <ImageGrid
       height={imagesGridHeight}
-      width={imagesGridWidth}
+      maxWidth={imagesGridMaxWidth}
       numberOfImgs={numberOfImgs}
       row={rowCol.row}
       col={rowCol.col}
     >
-      {images.map((img, index) => (
-        <ImageWrap>
-          <ImageItem src={img} />
-        </ImageWrap>
-      ))}
+      {numberOfImgs <= 5 &&
```

Comment:
try not to use hardcoded values right inside the code. If you need to use a certain constant value, I suggest adding constant variables for them with explicit names. Standard convention for it would be UPPERCASE + underscore, e.g.:

`const MAX_PREVIEW_NUM = 5`

Reply:
name is up to you, just an example